src/simulations/sample.jl

```diff
@@ -35,16 +35,18 @@ function sample!(
     add_newids!(model, org_ids, newids)
 end
 
-"Used in sample!"
+#Used in sample!
 function add_newids!(model, org_ids, newids)
     n = nextid(model)
+    count_newids = counter(newids)
```

Comment:
Where does `counter` come from...?

Reply:
it comes from `DataStructures.jl`

Reply:
aaaah maybe write a comment about htis in the source...?

Reply:
done!